### ISSUE TITLE
enhance accessibility

### ISSUE THREAD

**Issue by ericafenyo** [ericafenyo/quickheadlines#3]
* Use W3C recommended color ratios for both text and icons.
    * Small text should have a contrast ratio of at least 4.5:1 against its background.
    * Large text (at 14 pt bold/18 pt regular and up) should have a contrast ratio of at least 3:1 against its background.

* Layout and typography:
  * Extend Touch targets padding and spacing.
  * Use short descriptive text for accessibility text.
  * Use action verbs for "icons button". Example `Edit` over `Pencil`.
  * Add content description attributes to all `clickable` and `focusable` elements.
